src/transformers/trainer.py

```diff
@@ -2235,6 +2235,8 @@ def _load_best_model(self):
                     # If the model is on the GPU, it still works!
                     # workaround for FSDP bug https://github.com/pytorch/pytorch/issues/82963
                     # which takes *args instead of **kwargs
+                    if getattr(model, "is_loaded_in_8bit", False) and getattr(model, "is_8bit_serializable", False):
+                        model.cuda()
```

Comment:
Can you move this before the comment? Thanks!

Reply:
okay, thanks for reminding